examples/chip-tool/commands/common/Command.h

```diff
@@ -90,9 +91,23 @@ class Command
         ::chip::Inet::InterfaceId interfaceId;
     };
 
+    /**
+     * @brief
+     *   Encapsulates key objects in the CHIP stack that need continued
+     *   access, so wrapping it in here makes it nice and compactly encapsulated.
+     */
+    struct ExecutionContext
+    {
+        ChipDeviceCommissioner * Commissioner;
```

Comment:
nit: we generally have members lowercase (and class/struct names uppercase)

Reply:
I believe the prevailing style is

```
struct ExecutionContext {
  ChipDeviceCommissioner * commissioner;
};
```

and

```
class ExecutionContext {
 private:
  ChipDeviceCommissioner * mCommissioner;
};
```


For a capitalized identifier I'd expect it to be a type, not a variable.